frontend/probloom/Dockerfile

```
@@ -0,0 +1,22 @@
+FROM node:16
+
+WORKDIR /usr/app
+
+COPY package.json .
+COPY tsconfig.json .
+COPY yarn.lock .
+COPY src/ src/
+COPY public/ public/
+COPY build/ build/
+# RUN yarn install
+# RUN yarn build
```

This part cannot be run in our EC2 instance. I think it is because of our limited computing resource, but there might be other reasons.

If we somehow are able to build our front-end app, these lines should be changed as follows:
```suggestion
RUN yarn install
RUN yarn build
```